catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/SegmentHandler.java

```diff
@@ -70,17 +73,31 @@ private void handleTreValues(Metacard metacard, NitfAttribute attribute, Tre tre
   }
 
   private <T> void handleValue(Metacard metacard, NitfAttribute attribute, T segment) {
-
     Function<T, Serializable> accessor = attribute.getAccessorFunction();
-    Serializable value = accessor.apply(segment);
+
+    Serializable value;
+    try {
+      value = accessor.apply(segment);
+    } catch (NitfParsingException e) {
+      LOGGER.debug(
+          "Error accessing NITF attribute value. Skipping attribute [{}] on Metacard with ID [{}]",
+          attribute.getLongName(),
+          metacard.getId(),
+          e);
+
+      if (!attribute.getLongName().startsWith(ExtNitfUtility.EXT_NITF_PREFIX)) {
```

Comment:
❓ Wonder if we shouldn't add a `isExtAttribute(NitfAttribute)` on the `ExtNitfUtility` class instead of having to have this conditional logic here.